homeassistant/components/goalzero/manifest.json

```diff
@@ -8,5 +8,6 @@
     {"hostname": "yeti*"}
   ],
   "codeowners": ["@tkdrob"],
+  "quality_scale": "platinum",
```

Comment:
Complete tests are missing for this integration. There might be more things missing. Please copy the bullet list of requirements for the different levels and make a comment on status for each bullet in a PR comment.

https://developers.home-assistant.io/docs/integration_quality_scale_index

Reply:
```suggestion
  "quality_scale": "silver",
```